code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -137,6 +137,17 @@
 /obj/item/weapon/reagent_containers/spray/waterflower/attack_self(mob/user) //Don't allow changing how much the flower sprays
 	return
 
+//syndicate water flower
+/obj/item/weapon/reagent_containers/spray/syndiwaterflower

```

Comment:
You're right of course. But being a child of the normal waterflower does remove the ability to change the reagent output rate.